### PR TITLE
Fix config is not init yet in setup

### PR DESCRIPTION
**Pull request by ZobelOmry**
## Context

Take into account that setup could happen when there is no local storage config yet.
Fix this exception 
image

## Tests

- [ ]  Will be tested on marketly when in prod

### CODE REVIEW

**Comment by ZobelOmry** [comment]
@DvirMalka you alive?

File: dist/development/strigo.sdk.js [hunk 1]
```diff
@@ -13446,7 +13445,7 @@ ${JSON.stringify(parsedContext)}` : "");
       const mainDiv = generatePageStructure();
       const academyPanelFrame = appendIFrame({
         parentElement: mainDiv,
-        url: generateStrigoIframeURL(config),
+        url: generateStrigoIframeURL(config, false),
```
Comment (DvirMalka) [inline dist/development/strigo.sdk.js:13448]: why was this changed? weird

Comment (ZobelOmry) [reply dist/development/strigo.sdk.js:13448]: IDK

Comment (DvirMalka) [reply dist/development/strigo.sdk.js:13448]: @ZobelOmry are you rebased on the latest master + build after rebase?

Comment (ZobelOmry) [reply dist/development/strigo.sdk.js:13448]: looks ok
It's compatible with this https://github.com/strigo/strigo-sdk/blob/41c711786271b66e7f3ffb114320e5db183752a3/src/modules/widgets/iframe.ts#L80

Comment (ZobelOmry) [reply dist/development/strigo.sdk.js:13448]: maybe it wasn't built before @DvirMalka

Comment (DvirMalka) [reply dist/development/strigo.sdk.js:13448]: sababa. Just check with @hidday everything is as it should be (looks ok to me too)

**Review by DvirMalka — APPROVED** [review]

File: dist/development/strigo.sdk.js (hunk 1 above)
Comment (hidday) [reply dist/development/strigo.sdk.js:13448]: Wait
what

Comment (hidday) [reply dist/development/strigo.sdk.js:13448]: in the iFrame option (which is not really supported) - we just added it for consistency (ESLint hated it)

Comment (DvirMalka) [reply dist/development/strigo.sdk.js:13448]: @hidday that's not the iFrame option. that's Strigo Iframe (strigo-app panel) which we did change yesterday...

Comment (hidday) [reply dist/development/strigo.sdk.js:13448]: where is it coming from? I don't see it int the source TS file

Comment (DvirMalka) [reply dist/development/strigo.sdk.js:13448]: (to include the preview mode option)

Comment (hidday) [reply dist/development/strigo.sdk.js:13448]: OK cool

Comment (DvirMalka) [reply dist/development/strigo.sdk.js:13448]: @ZobelOmry MERGE IT

Comment (hidday) [reply dist/development/strigo.sdk.js:13448]: @ZobelOmry Ping me for 5 minutes verification of this

Comment (hidday) [reply dist/development/strigo.sdk.js:13448]: o shelo

Comment (ZobelOmry) [reply dist/development/strigo.sdk.js:13448]: already pinged
